src/platforms/esp32/components/avm_builtins/spi_driver.c

```diff
@@ -63,6 +63,9 @@
 #elif CONFIG_IDF_TARGET_ESP32C3
 // only one user SPI bus, no VSPI
 #define HSPI_HOST   SPI2_HOST
+#elif (ESP_IDF_VERSION >= ESP_IDF_VERSION_VAL(5, 1, 0)) && CONFIG_IDF_TARGET_ESP32C6
```

Comment:
The IDF version check shouldn't be needed, attempting to use `idf.py set-target esp32-c6` on earlier versions will fail. So any attempt to build for C6 models with older versions will fail.

Reply:
right, but some constants are even not defined, leading to an error, so we need to check that.